packages/messaging/src/controllers/base-controller.ts

```diff
@@ -238,7 +238,12 @@ export abstract class BaseController
    */
   private async deleteTokenFromDB(token: string): Promise<void> {
     const tokenDetails = await this.tokenDetailsModel.deleteToken(token);
-    await this.subscriptionManager.deleteToken(this.services, tokenDetails);
+    try {
+      await this.subscriptionManager.deleteToken(this.services, tokenDetails);
+    } catch (e) {
```

Comment:
Is it possible to check the response code for this particular error, rather than catching all errors?

Reply:
The response is just "500 Internal Error" so it's not really distinguishable from other errors :man_shrugging: